src/files/BrsFile.spec.ts

```diff
@@ -2877,6 +2877,215 @@ describe('BrsFile', () => {
+                `);
+                await testTranspile(`
+                    import "pkg:/source/types.bs"
+                    alias sf = someFunc
+
+                    namespace alpha
+                        sub someFunc()
+                            sf()
+                        end sub
+                    end namespace
+                `, `
+                    'import "pkg:/source/types.bs"
+                    'alias sf = someFunc
+                    sub alpha_someFunc()
+                        someFunc()
+                    end sub
+                `);
+            });
+
+            it('replaces aliased consts', async () => {
+                program.setFile('source/types.bs', `
+                    const PI = 3.14
+                `);
+                await testTranspile(`
+                    import "pkg:/source/types.bs"
+                    alias p = PI
+
+                    namespace alpha
+                        function pi() as string
+                            return "apple"
+                        end function
+
+                        sub printPi()
+                            print p
+                        end sub
+                    end namespace
+                `, `
+                    'import "pkg:/source/types.bs"
+                    'alias p = PI
+                    function alpha_pi() as string
+                        return "apple"
+                    end function
+
+                    sub alpha_printPi()
+                        print 3.14
+                    end sub
+                `);
+            });
+
+            it('replaces aliased enums', async () => {
+                program.setFile('source/types.bs', `
+                    enum Direction
+                        north = "North"
+                        south = "South"
+                    end enum
+                `);
+                await testTranspile(`
+                    import "pkg:/source/types.bs"
+                    alias dir = Direction
+                    alias dirN = Direction.north
+
+                    namespace alpha
+                        function Direction() as string
+                            return "apple"
+                        end function
+
+                        sub printDir()
+                            print dir.north
+                            print dirN
+                        end sub
+                    end namespace
+                `, `
+                    'import "pkg:/source/types.bs"
+                    'alias dir = Direction
+                    'alias dirN = Direction.north
+                    function alpha_Direction() as string
+                        return "apple"
+                    end function
+
+                    sub alpha_printDir()
+                        print "North"
+                        print "North"
+                    end sub
+                `);
+            });
+
+            it('can deep alias a namespaced thing', async () => {
```

Comment:
Can you add a test for aliased class? And make sure we can `new` the class using the alias name? And that it transpiles properly?
i.e.:
```
alias Person2 = Person
sub test()
     dude = new Person2()
end sub
```